share/completions/isatty.fish

```diff
@@ -1 +1,5 @@
-complete -c isatty -x -k -a "stdin stdout stderr"
+complete -c isatty -x 
+complete -c isatty -k -a "(string replace /dev/fd/ '' /dev/fd/*)"
```

Comment:
This will error out if `/dev/fd` doesn't exist. Which it won't on non-linux systems, if I recall correctly.

Reply:
FYI it exists here on macOS, and I think it's been in BSD forever but it should still be fixed for cases where it does not exist.

```shell
ls -l /dev/fd
total 0
crw--w----   1 floam  tty    0x10000004 Mar 28 11:35 0
crw--w----   1 floam  tty    0x10000004 Mar 28 11:35 1
crw--w----   1 floam  tty    0x10000004 Mar 28 11:35 2
drw-r-----  55 floam  staff        1760 Mar 28 11:34 3/
dr--r--r--   1 root   wheel           0 Mar 16 15:55 4/
```